Automattic/wp-calypso
# State: Refactor term addition as receive / remove action pair

Related: #6137

This pull request seeks to refactor `addTerm` to using generic `TERMS_RECEIVE` and `TERM_REMOVE` actions instead of add-specific action types.

__Testing instructions:__

Ensure Mocha tests pass:

```
npm run test-client client/state/terms
```

/cc @timmyc 

Test live: https://calypso.live/?branch=update/state-terms-receive
